cosmos/operators/local.py

```diff
@@ -500,9 +502,10 @@ def run_command(
                 self.store_freshness_json(tmp_project_dir, context)
                 self.store_compiled_sql(tmp_project_dir, context)
                 self.upload_compiled_sql(tmp_project_dir, context)
-                self.handle_exception(result)
                 if self.callback:
-                    self.callback(tmp_project_dir)
+                    self.callback_args.update(context)
```

Comment:
It would be worth to set "context" as a kwarg to `callback_args`
```
self.callback_args["context"] = context
```
Making the interface clearer

Reply:
done so, thanks.